### PR TITLE
Processing Handlers

### PR DESCRIPTION
**Pull request by satchit24**
Fixes #106

### CODE REVIEW

File: src/processing/handlers/ObjectHandler.java [hunk 1]
```diff
@@ -25,24 +24,107 @@
...
+				case ROTATE:
+					
+					//Angle of rotation of the object is obtain
+					Angle angleOfRotation = ((
+							RotateOperation)
+							boardOperationType).
+							getAngle();
+					
+					//Perform rotate operation on the BoardObject
+					ParameterizedOperationsUtil.rotationUtil(
```
Comment (himanshu2047) [inline src/processing/handlers/ObjectHandler.java]: call rotate instead of rotationUtil

File: src/processing/handlers/ObjectHandler.java [hunk 2]
```diff
@@ -25,24 +24,107 @@
...
+				case COLOR_CHANGE:
+					
+					//Intensity is of the object is obtain
+					Intensity newIntensity = ((
+							ColorChangeOperation)
+							boardOperationType).
+							getIntensity();
+					
+					//Perform color change operation on the BoardObject
+					ParameterizedOperationsUtil.colorChangeUtil(
```
Comment (himanshu2047) [inline src/processing/handlers/ObjectHandler.java]: same here

File: src/processing/handlers/ObjectHandler.java [hunk 3]
```diff
@@ -25,24 +24,107 @@
...
+					ParameterizedOperationsUtil.rotationUtil(
+							boardObject,
+							newUserId,
+							angleOfRotation
+					);
+					break;
+				case COLOR_CHANGE:
+					
+					//Intensity is of the object is obtain
+					Intensity newIntensity = ((
```
Comment (himanshu2047) [inline src/processing/handlers/ObjectHandler.java]: wrong format

**Review by himanshu2047 — CHANGES_REQUESTED** [review]

File: src/processing/handlers/ObjectHandler.java [hunk 4]
```diff
@@ -25,24 +24,107 @@
...
+					break;
+				case DELETE:
+					
+					//Perform delete operation on the BoardObject
+					SelectDelete.delete(boardObject, newUserId);
+					break;
+				case ROTATE:
+					
+					//Angle of rotation of the object is obtain
+					Angle angleOfRotation = ((
```
Comment (himanshu2047) [inline src/processing/handlers/ObjectHandler.java]: wrong format, It should be like - 
Angle angleOfRotation = ((RotateOperation) boardOperationType)
                     .getAngle();

Comment (satchit24) [reply src/processing/handlers/ObjectHandler.java]: will change

File: src/processing/handlers/ObjectHandler.java (hunk 1 above)
Comment (satchit24) [reply src/processing/handlers/ObjectHandler.java]: noted

File: src/processing/handlers/ObjectHandler.java (hunk 3 above)
Comment (satchit24) [reply src/processing/handlers/ObjectHandler.java]: noted

File: src/processing/handlers/ObjectHandler.java (hunk 2 above)
Comment (satchit24) [reply src/processing/handlers/ObjectHandler.java]: noted

**Review by himanshu2047 — APPROVED** [review]

**Pull request merged**